graph/schema.graphqls

```diff
@@ -3,28 +3,39 @@ Copyright Contributors to the Open Cluster Management project.
 
 Search Query API.
 """
-schema { 
+schema {
   query: Query
+  subscription: Subscription
+}
+
+"""
+`Subscriptions` supported by the Search Query API.
```

Comment:
Avoid the word "supported". Maybe
"Subscriptions implemented by the Search Query API."